2020-07-25-CybricsCTF/xcorp/README.md

```diff
@@ -0,0 +1,53 @@
+# XCorp (network, 50p, 159 solved)
+
+In the task we get a [pcap](xcorp.pcap) to analyse.
+Networ Miner recovers [executable](main.exe) for us included in the pcap.
```

Comment:
There is no main.exe ; did you mean net10.exe ?

Reply:
Yes, that's the one :)